### DISCUSSION TITLE
Testing form

### DISCUSSION THREAD

**Discussion by EdorianDark** [tokio-rs/axum#361] [Q&A] [answered]
I wanted to test a form handler similar to [the example](https://github.com/tokio-rs/axum/blob/main/examples/form/src/main.rs). But I struggled until I found out about ``.header("Content-Type", "application/x-www-form-urlencoded")``.
Would it make sense to add a unit test to the example?
I could make a pull request.

**Comment by davidpdrsn** [comment, marked as answer]
It's mentioned in the docs https://docs.rs/axum/0.2.6/axum/extract/struct.Form.html

**Reply by EdorianDark** [reply]
Oh, thanks.
Axum feels really nice to use!
